ops/terraform/services/pipeline/main.tf

```diff
@@ -206,5 +206,6 @@ module "bfd_pipeline_slis" {
   source          = "./modules/bfd_pipeline_slis"
   account_id      = local.account_id
   aws_kms_key_arn = local.kms_key_id
```

Comment:
Are `aws_kms_key_arn` and `aws_kms_key_id` both meant to be `local.aws_kms_key_id` ?